declarations/LoveScout24.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "LoveScout24",
+  "documents": {
+    "Terms of Service": {

```

Comment:
Wrong type of document, fixed in https://github.com/OpenTermsArchive/services-dating/pull/2/commits/1e1724137aeb786f52f934767eda3026c9307d1c